src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -466,6 +496,58 @@ internal void Update(double number)
                             sw.SpinOnce();
                         }
 
+                        break;
+                    }
+
+                case AggregationType.ExponentialHistogram:
+                    {
+                        var sw = default(SpinWait);
+                        while (true)
+                        {
+                            if (Interlocked.Exchange(ref this.exponentialBucketHistogram.IsCriticalSectionOccupied, 1) == 0)
+                            {
```

Comment:
Call `this.exponentialHistogram.Record(value)`?

Reply:
My plan was to do this in the next PR.